charmcraft.yaml

```diff
@@ -8,13 +8,13 @@ description: |
   Observability Stack (COS).
 subordinate: false
 
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
+parts:
+  charm:
+    plugin: charm
+    source: .
+
+platforms:
+  ubuntu@22.04:amd64:
```

Comment:
Is there a reason why we are not adding ubuntu@20.04:amd64 and ubuntu@24.04:amd64 ?

Reply:
This charm has only supported jammy from release. We could probably add noble support, but I'm not sure if we'd hit issues with dependencies - we can maybe do that in a new ticket?

Reply:
Yes, it wasn't there before

Reply:
Ok. Thanks for the info